### PR TITLE
modal overflow issue

### PR DESCRIPTION
**Pull request by rakeshyc**
This will fix the overflow of modal out of viewport.

### CODE REVIEW

File: scss/product/components/_modal.scss [hunk 1]
```diff
@@ -81,7 +81,9 @@
 }
 // Actual modal
 .modal-content {
-  max-height: 100vh;
+  // for avoiding overflow of modal
+  // as we are calling 8% margin for modal container we cant have height more than 84vh
+  max-height: 85vh;
```
Comment (is-ashish) [inline scss/product/components/_modal.scss:86]: 8% margin, then max-height should be `max-height: 84vh;` Right @rakeshyc

Comment (rakeshyc) [reply scss/product/components/_modal.scss:86]: will update @is-ashish

**Review by is-ashish — APPROVED** [review]
lgtm

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**